app/components/landscape-portrait-slider.js

```diff
@@ -1,32 +1,242 @@
 // https://github.com/EnCiv/undebate-ssp/issues/120
 
-import React, { useState, useEffect, useRef } from 'react'
+import React, { useState } from 'react'
 import { createUseStyles } from 'react-jss'
 import cx from 'classnames'
+import SvgDesktop from '../svgr/desktop'
+import SvgPortrait from '../svgr/portrait'
+import SvgRectangle from '../svgr/rectangle'
+import SvgExternalLink from '../svgr/external-link'
 
+const daLink = (
+    <a
+        style={{
+            textDecorationLine: 'none',
+            color: 'grey',
+            fontFamily: 'poppins',
+        }}
+        href={'https://cc.enciv.org/country:us/organization:ucla-student-accociation/office:usac-president/2021-05-07'}
+    >
+        District Attorney Election
+        <SvgExternalLink />
+    </a>
+)
+const portraitHeader = 'Tailored for every device and every layout'
```

Comment:
portraitHeader and paragraph should go in the story, and be passed as the portraitStatement property to this component. 

and now the Statement component is done, so you can merge with main and get it, and then uses that component, and pass these properties into that.